massa-client/src/cmds.rs

```diff
@@ -576,13 +576,14 @@ impl Command {
             }
 
             Command::get_filtered_sc_output_event => {
-                let p_list: [&str; 6] = [
+                let p_list: [&str; 7] = [
```

Comment:
Need to update the help at the top to match this new behavior

Reply:
Done